ncj/3dsmax/scripts/3dsmax.ps1

```diff
@@ -205,12 +205,7 @@ if (ParameterValueSet $renderPresetFile)
 mkdir -Force images > $null
 
 # Render
-$max_exec = "3dsmaxcmdio.exe"
-if ($env:3DSMAX_2018 -and (Test-Path "$env:3DSMAX_2018"))
-{
-    # New image
-    $max_exec = "${env:3DSMAX_2018}\3dsmaxcmdio.exe"
-}
+$max_exec = $env:3DSMAX_2018_EXEC
```

Comment:
will this work with older versions of Max?

Reply:
It will now 


---
In reply to: [186914187](https://github.com/Azure/BatchLabs-data/pull/16#discussion_r186914187) [](ancestors = 186914187)